book/creating-a-basic-website/configuring-image-formats.rst

```diff
@@ -84,3 +70,36 @@ To set the compression for all images you have to add following lines to your
 
 With the theme we got the container for our Twig-templates. That's what we'll
 do next: Writing awesome Twig files.
+
+Compression per format
+----------------------
+
+Quality, compression level as well as other options for `Imagine` can be defined
+per image format. When there is also a global configuration the values of the image format
+overrule the global ones. Let's define a compression for our image format.
+
+.. code-block:: xml
+
+        <format key="1200x800">
+            <meta>
+                <title lang="en">My header image format</title>
+                <title lang="de">Mein Header-Bildformat</title>
+            </meta>
+            <scale x="1200" y="800"/>
+            <options>
+                <option name="jpeg_quality">70</option>
+                <option name="png_compression_level">5</option>
+            </options>
+        </format>
+
+Custom image-format location
+----------------------------
+
+Often it's needed to store image-formats in files other than `%kernel.root_dir%/config/image-formats.xml`
+or the `ìmage-formats.xml` file of a theme. With the configuration `sulu_media.image_format_files` this is possible
+Just include the following lines in a config file (or prepend to the configuration with an extension).
```

Comment:
How does this actually work if it is set like that? I am afraid that it will override all existing paths, and only the two in this example will remain. I think there is something like a merge option in the symfony configuration component, which could help here.